docs/content/en/integrations/social-authentication.md

```diff
@@ -191,6 +191,26 @@ user, such as 'staff' or 'superuser'
 5.  Restart your Dojo, and you should now see a **Login with Azure AD**
     button on the login page which should *magically* work
 
+### Automatic Import of User-Groups
+To import groups from AzureAD users, the following env. var needs to be set:  
+
+    {{< highlight python >}}
+    DD_SOCIAL_AUTH_AZUREAD_TENANT_OAUTH2_GET_GROUPS=True
+    {{< /highlight >}}
+This allows to automatically limit the products a user can interact with.
```

Comment:
I don't think this setting does anything to limit the products a user can interact with. It just adds users to the groups copied from Azure AD.

Reply:
Is it better to understand now?

Reply:
I suggest:
"This will ensure the user is added to all the groups found in the Azure AD Token. Any missing groups will be created in Defect Dojo (unless filtered). This group synchronization allows for product access via groups to limit the products a user can interact with".